tools/makeboards.py

```diff
@@ -326,6 +326,8 @@ def MakeBoard(name, chip, vendor_name, product_name, vid, pid, pwr, boarddefine,
             BuildPSRAMCS(name)
             BuildPSRAM(name)
             BuildPSRAMFreq(name)
+        elif name == "datanoisetv_picoadk_v2":
+            BuildFreq(name, 400)
```

Comment:
Sorry, but again you still have 2 frequency menus being generated.  Line 323 and line 330.